docs/Experiments.rst

```diff
@@ -121,31 +121,31 @@ Accuracy
-|           | NDCG\ :sub:`10` | 0.78089  | 0.783013      | 0.796818 |
-+-----------+-----------------+----------+---------------+----------+
-| MS LTR    | NDCG\ :sub:`1`  | 0.483956 | 0.488649      | 0.524255 |
-|           +-----------------+----------+---------------+----------+
-|           | NDCG\ :sub:`3`  | 0.467951 | 0.473184      | 0.505327 |
-|           +-----------------+----------+---------------+----------+
-|           | NDCG\ :sub:`5`  | 0.472476 | 0.477438      | 0.510007 |
-|           +-----------------+----------+---------------+----------+
-|           | NDCG\ :sub:`10` | 0.492429 | 0.496967      | 0.527371 |
-+-----------+-----------------+----------+---------------+----------+
-| Expo      | AUC             | 0.756713 | 0.777777      | 0.777543 |
-+-----------+-----------------+----------+---------------+----------+
-| Allstate  | AUC             | 0.607201 | 0.609042      | 0.609167 |
-+-----------+-----------------+----------+---------------+----------+
++-----------+-----------------+----------+-------------------+--------------+
+| Data      | Metric          | xgboost  | xgboost\_hist     | LightGBM     |
++===========+=================+==========+===================+==============+
+| Higgs     | AUC             | 0.839593 | **0.845605**      | 0.845154     |
++-----------+-----------------+----------+-------------------+--------------+
+| Yahoo LTR | NDCG\ :sub:`1`  | 0.719748 | 0.720223          | **0.732466** |
+|           +-----------------+----------+-------------------+--------------+
+|           | NDCG\ :sub:`3`  | 0.717813 | 0.721519          | **0.738048** |
+|           +-----------------+----------+-------------------+--------------+
+|           | NDCG\ :sub:`5`  | 0.737849 | 0.739904          | **0.756548** |
+|           +-----------------+----------+-------------------+--------------+
+|           | NDCG\ :sub:`10` | 0.78089  | 0.783013          | **0.796818** |
++-----------+-----------------+----------+-------------------+--------------+
+| MS LTR    | NDCG\ :sub:`1`  | 0.483956 | 0.488649          | **0.524255** |
+|           +-----------------+----------+-------------------+--------------+
+|           | NDCG\ :sub:`3`  | 0.467951 | 0.473184          | **0.505327** |
+|           +-----------------+----------+-------------------+--------------+
+|           | NDCG\ :sub:`5`  | 0.472476 | 0.477438          | **0.510007** |
+|           +-----------------+----------+-------------------+--------------+
+|           | NDCG\ :sub:`10` | 0.492429 | 0.496967          | **0.527371** |
++-----------+-----------------+----------+-------------------+--------------+
+| Expo      | AUC             | 0.756713 | **0.777777**      | 0.777543     |
++-----------+-----------------+----------+-------------------+--------------+
+| Allstate  | AUC             | 0.607201 | 0.609042          | **0.609167** |
++-----------+-----------------+----------+-------------------+--------------+
```

Comment:
I thought it was valuable to visually differentiate the best performer on each problem / metric